### PR TITLE
fix book examples

### PR DESCRIPTION
**Pull request by helinwang**

### CODE REVIEW

File: recommender_system/README.en.md [hunk 1]
```diff
@@ -83,11 +83,6 @@ We use the [MovieLens ml-1m](http://files.grouplens.org/datasets/movielens/ml-1m
 
 `paddle.v2.datasets` package encapsulates multiple public datasets, including `cifar`, `imdb`, `mnist`, `moivelens` and `wmt14`, etc. There's no need for us to manually download and preprocess `MovieLens` dataset.
 
-```python
-# Run this block to show dataset's documentation
-help(paddle.v2.dataset.movielens)
```
Comment (helinwang) [inline recommender_system/README.en.md:88]: paddle need to be imported before using `paddle.v2.dataset.movielens`, but the introduction and code for paddle import and init is below. And importing paddle twice will cause jupytor error. So I am removing this line.

File: image_classification/README.en.md [hunk 2]
```diff
@@ -170,6 +170,8 @@ We must import and initialize PaddlePaddle (enable/disable GPU, set the number o
 ```python
 import sys
 import paddle.v2 as paddle
+from vgg import vgg_bn_drop
```
Comment (wangkuiyi) [inline image_classification/README.en.md:173]: 这个程序是谁写的？如果没有这个修改，应该跑不通？

File: recommender_system/README.en.md (hunk 1 above)
Comment (wangkuiyi) [reply recommender_system/README.en.md:88]: Great to know this! Is there the same problem in the Chinese version README.md?

File: understand_sentiment/README.en.md [hunk 3]
```diff
@@ -344,7 +344,7 @@ Finally, we can invoke `trainer.train` to start training:
 trainer.train(
     reader=train_reader,
     event_handler=event_handler,
-    feeding=feedig,
+    feeding=feeding,
```
Comment (wangkuiyi) [inline understand_sentiment/README.en.md:347]: 这是是谁写的？如果没有这个修改看上去也是跑不通的。

**Review by wangkuiyi — APPROVED** [review]
Great appreciation!

File: recommender_system/README.en.md (hunk 1 above)
Comment (helinwang) [reply recommender_system/README.en.md:88]: No, Chinese version is a little different. It imports paddle in the beginning.

**Pull request merged**